### PR TITLE
 Theming improvements

### PR DESCRIPTION
**Pull request by Jikstra**
- [x] Move ThemeSystem.md to new `/docs` folder
- [x] Introduce --theme argument which force updates the used theme
  - [x] update docs
- [x] --theme-watch now only watches whatever theme is loaded
- [x] move out default/light theme into own `/themes/light.json` file
- [x] make `npm start` and `npm run dev` pass arguments to electron process

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -77,20 +91,27 @@ function onReady (err, results) {
...
-    log.info('theme-watch activated: ', app.rc['theme-watch'])
-    if (fs.existsSync(app.rc['theme-watch'])) {
-      fs.watchFile(app.rc['theme-watch'], (curr, prev) => {
+    log.info('theme-watch: activated', app.rc['theme-watch'])
+    if(app.rc['theme']) {
+      fs.watchFile(app.rc['theme'], (curr, prev) => {
         if (curr.mtime !== prev.mtime) {
           log.info('theme-watch: File changed reloading theme data')
+          updateTheme()
           // TODO make this more safe or disable when merging
```
Comment (Simon-Laux) [inline src/main/index.js]: you can remove this line, it was there because I first tried loading the json with `require`

File: themes/light.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "bgImagePath": "../images/background_light.svg",
+  "bgPrimary": "#fff",
+  "bgSecondary": "#f5f5f5",
+  "accentColor": "#2090ea",
+  "textPrimary": "#010101",
+  "textSecondary": "#62656a",
+  "ovalButtonBg": "#415e6b",
+  "ovalButtonText": "#fff",
+  "bgChatView": "lime",
```
Comment (Simon-Laux) [inline themes/light.json]: change this to `"bgChatView": "#e6dcd4"` I forgot to do that lime looks really ugly.

**Comment by Simon-Laux** [comment]
I would move "fix at least some light theme bugs #924" to a dedicated pr, where we can also add some of the missing pieces described in the todo section of the documentation.

File: themes/darkpurple.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "meta_description":"A clean theme that highlights our purple nature",
```
Comment (Simon-Laux) [inline themes/darkpurple.json]: maybe it's too early to add metadata... I used it as comment here, because vanilla json supports no comments.

**Comment by Simon-Laux** [comment]
badge change
above new, bellow old

**Comment by Jikstra** [comment]
> I would move fix at least some light theme bugs #924 to a dedicated pr, where we can also add some of the missing pieces described in the todo section of the documentation.

I'm very fine with this.

**Comment by Simon-Laux** [comment]
The line was a comment warning of script injection because I tried to use require to load the json at the time

**Pull request merged**